themes/digital.gov/layouts/partials/core/touchpoints_feedback.html

```diff
@@ -0,0 +1,7 @@
+<div id="touchpoints_feedback">
+  <p>Was this page useful?</p>
```

Comment:
@jeremyzilar - try replacing the contents of the #touchpoints_feedack div with

`<script src="https://touchpoints-staging.app.cloud.gov/touchpoints/eaf1500b/js" async></script>`

Reply:
the CSS will be wonky, but we can start to get a sense of the feel and interaction

Reply:
![include](https://user-images.githubusercontent.com/67984/76467495-fa1ac200-63a6-11ea-97c3-a2b1417908c7.png)

Reply:
![rough-response](https://user-images.githubusercontent.com/67984/76468880-44517280-63aa-11ea-9b4c-a4161647687b.png)

Reply:
obviously, these need work. but sharing early.

Reply:
![q](https://user-images.githubusercontent.com/67984/76471677-95646500-63b0-11ea-870a-53374bfd8f90.png)

Reply:
![a](https://user-images.githubusercontent.com/67984/76471688-a0b79080-63b0-11ea-8164-df34a51eac7b.png)

Reply:
@jeremyzilar - still a slight variance in the bg color. I'll check it out.

But quite a bit closer, style-wise.